frontend-student/src/context/authContext.jsx

```
@@ -51,8 +50,9 @@ export const AuthProvider = ({ children }) => {
   const LoadingSpinner = () => {
     return (
       <div>
-        loading
-        <Spinner show={true} />
+        Loading...
+        {/* Add a simple spinner here if needed, like a rotating circle */}
+        <div className="animate-spin h-8 w-8 border-4 border-t-4 border-blue-500 rounded-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)